app/assets/stylesheets/nfg_ui/network_for_good/admin/color_variations/nfg_theme/_buttons.scss

```diff
@@ -1,10 +1,36 @@
 @each $color, $value in $colors {
   .#{$color} {
-    .btn-primary { @include button-variant($value, $value); }
+    .btn-primary {
+      @include button-variant($value, $value);
+      color: $white !important;
+      &:focus,
+      &.focus {
+        box-shadow: 0 0 0 $input-btn-focus-width transparentize($value, 0.8);
+      }
+      &:not(:disabled):not(.disabled):active,
+      &:not(:disabled):not(.disabled).active,
+      .show > &.dropdown-toggle {
+        box-shadow: 0 0 0 $input-btn-focus-width transparentize($value, 0.8);
+        &:focus { box-shadow: 0 0 0 $input-btn-focus-width transparentize($value, 0.8); }
+      }
+    }
+    .btn-link {
+      &:focus,
+      &.focus {
+        box-shadow: 0 0 0 $input-btn-focus-width transparentize($value, 0.8);
+      }
+      &:not(:disabled):not(.disabled):active,
+      &:not(:disabled):not(.disabled).active,
+      .show > &.dropdown-toggle {
+        box-shadow: 0 0 0 $input-btn-focus-width transparentize($value, 0.8);
+        &:focus { box-shadow: 0 0 0 $input-btn-focus-width transparentize($value, 0.8); }
+      }
+    }
     .show > .btn-primary.dropdown-toggle {
       background-color: darken($value, 10%);
       border-color: darken($value, 10%);
+      &:focus { box-shadow: 0 0 0 $input-btn-focus-width transparentize($value, 0.8); }
     }
-    .btn-outline-secondary, .btn-link { color: $value; }
+    .btn-secondary, .btn-outline-secondary, .btn-link { color: $value; }
   }
 }
```

Comment:
@kylejmorin we should double check that the display app includes disabled themed buttons for reference